src/components/OcTextarea.vue

```diff
@@ -158,9 +168,11 @@ export default {
       this.$emit("focus", value)
     },
     onKeyDown(e) {
-      if (e.keyCode === 13) {
+      e.keyEnter = e.key === "Enter";
```

Comment:
Hm.... personally in favor of not altering the event. Since `enterKey` doesn't exist, you could just create a `const` and use it in the `if` statement, that still improves the readability a bit.
I.e. like this:
```js
const enterKey = e.key === "Enter"
if (this.submitOnEnter && enterKey && !e.ctrlKey && !e.shiftKey) {
```

Reply:
looks fine for me ^^